examples/cra-kitchen-sink/src/stories/index.js

```diff
@@ -135,8 +135,8 @@ storiesOf('Button', module)
   )
   .add(
     'addons composition',
-    withInfo('see Notes panel for composition info')(
-      withNotes({ notes: 'Composition: Info(Notes())' })(context =>
+    withInfo({ text: 'see Notes panel for composition info' })(
```

Comment:
This is the example we show to users, do you think it makes sense like this?
I think the less verbose syntax:
`withInfo('see Notes panel for composition info')`
makes more sense tbh.

We should document these options somewhere, but the example should showcase the common use case here, I think.